src/DetailsView/components/details-view-command-bar.tsx

```diff
@@ -232,25 +234,25 @@ export class DetailsViewCommandBar extends React.Component<
     }
 
     private renderSaveAssessmentButton = (): JSX.Element | null => {
-        if (this.props.featureFlagStoreData.saveAndLoadAssessment) {
-            return this.props.switcherNavConfiguration.SaveAssessmentFactory({
-                ...this.props,
-            });
-        }
-        return null;
+        return this.props.switcherNavConfiguration.SaveAssessmentFactory({
```

Comment:
I think we can probably do the same flagged component bit here instead of moving it down a layer to the save assessment factory.

Additionally, after that we can change the type for this method and below to remove the `| null`.

Reply:
This has been addressed.